lbz/authz/__init__.py

```diff
@@ -0,0 +1,5 @@
+# compatibility with previous versions
```

Comment:
You don't have to write it directly - that is a common pattern to put imports like those into the `__init__.py`module.